app/src/androidTest/java/com/example/partyPlayer/InstrumentedTest.kt

```diff
@@ -10,11 +10,10 @@ import org.junit.Assert.*
 
 /**
  * Instrumented test, which will execute on an Android device.
- *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 @RunWith(AndroidJUnit4::class)
-class ExampleInstrumentedTest {
+class InstrumentedTest {
```

Comment:
Changed the name of the test files to be a little more visibly unique. (Both test files began with "Example")